### DISCUSSION TITLE
Plugins

### DISCUSSION THREAD

**Discussion by rahulbtc** [JushBJJ/Mr.-Ranedeer-AI-Tutor#45] [General]
I'm curious to know (as I did not find any answer yet), about the plugins and the version, that will be most helpful to make the most out of this prompt.
In this case, I am referring to the JSON format, V2.5 

**Consider the below cases:** 

1) In the search command: "  "search": "Search based on what the student specifies. *REQUIRES PLUGINS*",   "
-> In this case, is it better to use the "Search with bing" option? Or is there any plugin that will do the work? As of now, we can only select either of the above two options. 

2) in the visualise command: " "visualize": "Use plugins to visualize the content. *REQUIRES PLUGINS*" "
-> Which plugin will you recommend? 

3) General: Which version of ChatGPT should we use to make the most out of this prompt?
-> 1) ChatGPT4  2) ChatGPT4, Browse with bing (Internet access) 3) ChatGPT4 Plugins model

Thanks a lot for your time, and for the awesome project !

**Comment by JushBJJ** [comment]
1. I recommend `Search with Bing`
2. For visualization, I recommend `Wolfram Alpha` (For mathematical diagrams and latex enabled equations) and `Show Me` for visualised connections between different things.
3. It depends on what you want to learn, but Mr. Ranedeer works the best (without losing personality) in the default GPT-4 version. Wolfram still does work too for mathematics. It doesn't gobble up the token count too much.,

**Reply by rahulbtc** [reply]
I see.. Thanks a lot for your inputs.
